models/users.js

```diff
@@ -42,9 +42,17 @@ const addOrUpdate = async (userData, userId = null) => {
       const isNewUser = !user.data();
       // user exists update user
       if (user.data()) {
+        // remove id field if exist in fetched data or profileDiff
+        const currentData = user.data();
+        if ("id" in currentData) {
```

Comment:
We should not do this because it will not have any effect on the data getting merged into the database